### PR TITLE
Unit Tests: Regex

### PR DESCRIPTION
**Pull request by codeling**
@raghunayyar @georgehrke My first attempt at some unit tests for calendar. Please go easy on me ;)

### CODE REVIEW

File: tests/unit/utility/regexTest.php [hunk 1]
```diff
@@ -23,40 +23,187 @@
...
 class RegexTest extends \PHPUnit_Framework_TestCase {
 
-	/**
-	 * Initialize the calendar object we are going to test
-	 */
-	protected function setup() {
-
+	protected function setUp() {
+		parent::setUp();
 	}
```
Comment (georgehrke) [inline tests/unit/utility/regexTest.php]: if you just call parent::setUp(), you should be able to remove this method completely

File: tests/unit/utility/regexTest.php [hunk 2]
```diff
@@ -23,40 +23,187 @@
...
 
 
 	public function testURI() {
+		// URI not used anywhere in the code?
+		$this->assertSame( preg_match(RegexUtility::URI, "a"), 1 );
+		$this->assertSame( preg_match(RegexUtility::URI, "areallylongURIcontaininglowercaseUPPERCASEAndNumbers0123456789"), 1 );
+
 
+		$this->assertSame( preg_match(RegexUtility::URI, "" ), 0 );
+		//$this->assertSame( preg_match(RegexUtility::URI, "invalidURI!)/§$=§§"), 0 );
```
Comment (georgehrke) [inline tests/unit/utility/regexTest.php]: Why did you comment out this line? Is the URI regex broken?

**Comment by georgehrke** [comment]
Looks good otherwise, :+1: once the mentioned issues are fixed

**Comment by georgehrke** [comment]
Thx for this contribution. Looking forward to seeing more :)

**Comment by georgehrke** [comment]
Can you please squash these two commits? Thx :)
http://push.cwcon.org/learn/squashing-commits.html

**Comment by codeling** [comment]
@georgehrke done

**Comment by georgehrke** [comment]
thx :blush:

**Pull request merged**